edxval/api.py

```diff
@@ -974,6 +974,15 @@ def create_transcripts_xml(video_id, video_el, resource_fs, static_dir):
     if video_transcripts.exists():
         transcripts_el = SubElement(video_el, 'transcripts')
 
+    # Create static directory based on the file system's subdirectory,
+    # falling back to default path in case of an error
+    try:
+        # File system should not start from /draft directory.
+        static_file_dir = combine(resource_fs._sub_dir.split('/')[1], static_dir)  # pylint: disable=protected-access
+    except Exception:  # pylint: disable=broad-except
+        logger.exception("VAL Transcript Export: Error creating static directory path for video {}".format(video_id))
```

Comment:
Would it be useful to include anything like `static_dir` or `resource_fs` in this message?